pyanaconda/packaging/dnfpayload.py

```diff
@@ -555,11 +555,14 @@ def spaceRequired(self):
             if key.startswith('/') and ((root_mpoint + new_key) not in valid_points):
                 valid_points[root_mpoint + new_key] = val.format.freeSpaceEstimate(val.size)
 
-        m_points = _pick_mpoint(valid_points, download_size, size)
-        if not m_points or m_points == root_mpoint:
+        m_point = _pick_mpoint(valid_points, download_size, size)
+        if not m_point or m_point == root_mpoint:
             # download and install to the same mount point
             size = size + download_size
-        log.debug("Instalation space required %s for mpoints %s", size, m_points)
+            log.debug("Install + download space required %s", size)
+        else:
+            log.debug("Download space required %s for mpoint %s (non-chroot)", download_size, m_point)
```

Comment:
I don't think so. The problem is that you can pick download location as '/' it's totally valid and it then downloads to ram where the stage 2 is loaded. (I hope I said it right). 
So or so when you get log "Download space required 650MiB for mpoint /" will you know it's stage 2 and not installed root after few year from now? I want to prevent confusion from logs.